ci/package.sh

```diff
@@ -260,7 +264,11 @@ if ${CI_WAIT_FOR} image_build $nginx_arg \
     -f $script_dir/nginx/Dockerfile $script_dir \
     > ${build_dir}/image.$INDEX_IMAGE.${INDEX_VERSION}.log
 then
-    trace  "Output from appsody index build" "${build_dir}/image.$INDEX_IMAGE.${INDEX_VERSION}.log"
+    if [ "${VERBOSE}" == "true" ]; then
```

Comment:
I think this is an unneeded check. The trace call handles verboseness and there is already a condition in place to output if a failure occurs.

Reply:
I wanted to output the location of the output file if VERBOSE was false and I couldn't do this in the trace function as trace is generic.